docs/1.5/extensions/mention.md

```diff
@@ -1,145 +0,0 @@
----
-layout: default
-title: Mention Parser
-description: The MentionParser makes it easy to parse shortened references like @colinodell and #123 to custom URLs
----
-
-# Mentions
-
-Although not a full extension, the `MentionParser` makes it easy to parse shortened mentions and references like `@colinodell` to a Twitter URL or `#123` to a GitHub issue URL.  You can create your own custom syntax by choosing the starting symbol, a regular expression to match against, and any custom URL generator you wish.  Pre-built mention parsers are also available for use.
```

Comment:
Let's make this a true extension in 1.5 instead :wink:

Reply:
Done. This is now a proper `MentionExtension`